pkg/certificate/manager.go

```diff
@@ -105,7 +105,10 @@ func (m *Manager) EnsureCertificate(certReq Request, ownerName string) (Certific
 	keyFile := filepath.Join(m.K0sVars.CertRootDir, fmt.Sprintf("%s.key", certReq.Name))
 	certFile := filepath.Join(m.K0sVars.CertRootDir, fmt.Sprintf("%s.crt", certReq.Name))
 
-	uid, _ := users.GetUID(ownerName)
+	uid, err := users.GetUID(ownerName)
+	if err != nil {
+		logrus.Warnf("failed to get uid for %s, using uid 0: %v", ownerName, err)
```

Comment:
There are some os.Chown calls down the line in this function which ignore errors, as well. Do we want to log those, too?

Reply:
That's a good point. I've set these to Warning, I don't feel very strongly though, so if you want me to make these an error that's fine.

I'm not quite sure if it makes more sense to have these as warnings or errors. Anyway there it is.